appeals/web/src/server/app/app.controller.js

```diff
@@ -30,9 +30,9 @@ export const viewHomepage = async (request, response) => {
 			case config.referenceData.appeals.legalGroupId:
 			case config.referenceData.appeals.caseOfficerGroupId:
 			case config.referenceData.appeals.inspectorGroupId:
-				return response.redirect('/appeals-service/appeals-list');
-
+				return response.redirect('/appeals-service/personal-list');
```

Comment:
This will only happen if the current logged in user belongs to a single group. Maybe this logic should be updated?

Reply:
this has been updated now. If the user is a case officer or inspector they will be redirected, otherwise they'll go to the default landing page